build/targets/KoreBuild.Common.targets

```diff
@@ -53,6 +37,28 @@ as they are subject to change.
 
 <!--
 ###################################################################
+Targets: UseVolatileFeed
+
+Replaces $(RepositoryRoot)\NuGet.config feeds.
+###################################################################
+-->
+  <ItemGroup>
+    <UpdateFeeds Include="AspNetCore" Value="$(NUGET_VOLATILE_FEED_AspNetCore)" Condition="'$(NUGET_VOLATILE_FEED_AspNetCore)' != ''" />
```

Comment:
This is not the only feed we modify (although it's the one we most commonly change). Should be able to call `Environment.GetEnvironmentVariables` + StartsWith to filter